db/schema.rb

```diff
@@ -182,6 +190,13 @@
   add_index "paths", ["category_id"], name: "index_paths_on_category_id", using: :btree
   add_index "paths", ["language_id"], name: "index_paths_on_language_id", using: :btree
 
+  create_table "relationships", force: true do |t|
+    t.integer  "follower_id"
+    t.integer  "followed_id"
+    t.datetime "created_at"
```

Comment:
Please check you db and migrations. Here yo have introduced several changes that should not be there.